src/scripts/common.js

```diff
@@ -54,3 +54,9 @@ export function toMicroDenom(denom) {
   }
   return lookup[denom] ? lookup[denom] : denom.toLowerCase()
 }
+
+export function removeUFromMicroDenom(denom) {
```

Comment:
should this be done in the API?

do we have uATOM's anywhere in the FE?

Reply:
Sorry, forgot to remove. This is from the previous approach, `selectMostRelevantReward`

Reply:
Oops, just noticed this is actually needed. Could be done in the API though, although it would require changing stuff in the FE (filters for `uatoms)`.

The thing is that in e-Money these tokens are called `eCHF`, `eEUR`, and in Terra `uMNT` and so on... so in e-Money it makes sense to display the whole thing, but not in Terra (since we are not displaying the micro unit).

In e-Money these ones should be called `ueCHF` actually. I don't know why they have it like that. 

I think it is better to wait for transaction standardization, since doing this in the API would invove removing filters in the FE

Reply:
amazing function name, for the record.